src/appleseed/renderer/modeling/bssrdf/sss.h

```diff
@@ -183,24 +195,23 @@ double normalized_diffusion_max_radius(
 
 
 //
-// Attenuation sampling.
+// Dipole sampling.
 //
 // Reference:
 //
-//   Volumetric Path Tracing, Steve Marschner, section 3.
-//   http://www.cs.cornell.edu/courses/cs6630/2012sp/notes/09volpath.pdf
+//   A Practical Model for Subsurface Light Transport
+//   https://graphics.stanford.edu/papers/bssrdf/bssrdf.pdf
```

Comment:
Was the Volumetric Path Tracing reference relevant / useful / interesting? If so, it might make sense to keep it.